index.js

```diff
@@ -101,7 +102,8 @@ async function terraform(terraformDirectory, args) {
   }
 
   core.startGroup('Run terraform init');
-  const tfi = await terraform(terraformDirectory, ['init']);
+  let tfi_args = !terraformBackend ? ['init'] : ['init', '-backend-config=' + terraformBackend];
+  const tfi = await terraform(terraformDirectory, tfi_args);
```

Comment:
Perhaps something like this?

```suggestion
  let tfi_args = !terraformBackend ? [] : ['-backend-config=' + terraformBackend];
  const tfi = await terraform(terraformDirectory, ['init'].concat(tfi_args));
```